types/wordpress__components/combobox-control/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentType, ReactNode } from 'react';
+
+declare namespace ComboboxControl {
+    interface Props {
+        value: string | number;
```

Comment:
Here the `value` prop is allowed to be a number, but the value for each element in `options` must be a string.
[We're using `===` to find which the current option is](https://github.com/WordPress/gutenberg/blob/0135921bcb72ceec629fa8f9e755891afab747b9/packages/components/src/combobox-control/index.js#L58), so it seems like passing a number to `value` is always going to be a bug.

Are there instances you know of where `value` is a number? Maybe some existing bugs where you don't want to introduce a type error as soon as this type annotation package updates?

Reply:
Yeah, I wasn't sure about that. I saw that https://github.com/Automattic/wp-calypso/pull/59876 passed a `string | number` to this prop so I thought that maybe it was supposed to include numbers, but maybe that's just a type error. I'll comment there.